pkg/services/ldap/service/ldap.go

```diff
@@ -99,6 +99,34 @@ func (s *LDAPImpl) Reload(ctx context.Context, settings models.SSOSettings) erro
 		return err
 	}
 
+	// calculate MinTLSVersionID and TLSCipherIDs from input text values
+	// also initialize Timeout and OrgID from group mappings with default values if they are not configured
+	for _, server := range ldapCfg.Servers {
+		if server.MinTLSVersion != "" {
+			server.MinTLSVersionID, err = util.TlsNameToVersion(server.MinTLSVersion)
+			if err != nil {
+				s.log.Error("failed to set min TLS version, ignoring", "err", err)
```

Comment:
i'd add the server host to the output for a more detailed message

Reply:
Good one, thanks. I added it now.